### PR TITLE
FPL-68 added information displayed on the evidence tab based on form input

### PR DESCRIPTION
**Pull request by kurt-uk**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPL-68

### Change description ###
FPL-68 added information displayed on the evidence tab based on what is selected in the annex documents part of the form. if no information is entered, the title is displayed but the box is empty

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Comment by kurt-uk** [comment]
3570988-4k-wallpaper-space

File: e2e/paths/uploadDocuments_test.js [hunk 1]
```diff
@@ -7,19 +7,25 @@ Before((I, caseViewPage) => {
   caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
 });
 
-Scenario('Selecting to follow for a document produces a textarea', (I, uploadDocumentsPage) => {
-  I.selectOption(uploadDocumentsPage.fields.socialWorkChronologyStatus, 'To follow');
-  I.fillField(uploadDocumentsPage.fields.socialWorkChronologyReason, 'mock reason');
+Scenario('Completing half the Annex documents form by selecting social work chronology document to follow in the c110a application', (I, uploadDocumentsPage, caseViewPage) => {
```
Comment (swalker125) [inline e2e/paths/uploadDocuments_test.js]: I would just take out the first half of the question so it reads something like 'selecting social work chronology document to follow in the c110a application'

File: e2e/paths/uploadDocuments_test.js [hunk 2]
```diff
@@ -7,19 +7,25 @@ Before((I, caseViewPage) => {
...
+  uploadDocumentsPage.socialWorkChronologyToFollow();
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
   I.seeEventSubmissionConfirmation(config.applicationActions.uploadDocuments);
+  caseViewPage.selectTab('evidence');
+  I.seeDocument('Social work chronology', '', 'To follow', 'mock reason');
 });
 
-Scenario('All documents are able to be uploaded', (I, uploadDocumentsPage) => {
-  uploadDocumentsPage.uploadSocialWorkChronology(config.testFile);
+Scenario('Complete the Annex documents part of the form by uploading all files in the c110a application', (I, uploadDocumentsPage, caseViewPage) => {
```
Comment (swalker125) [inline e2e/paths/uploadDocuments_test.js]: 'uploading all files in the c110a application'

something along those lines

File: e2e/pages/uploadDocuments/uploadDocuments.js [hunk 3]
```diff
@@ -50,4 +44,9 @@ module.exports = {
     I.fillField(this.fields.otherDocumentsTitle_2, 'Document');
     I.attachFile(this.documents.otherDocuments_2, file);
   },
+
+  socialWorkChronologyToFollow() {
```
Comment (swalker125) [inline e2e/pages/uploadDocuments/uploadDocuments.js]: I think the function needs to be renamed something like 'selectingSocialWorkChronologyToFollow' that's very long though 🤔

**Comment by kurt-uk** [comment]
3570988-4k-wallpaper-space

File: e2e/steps_file.js [hunk 4]
```diff
@@ -31,5 +31,15 @@ module.exports = function () {
       this.click(link);
       this.seeCurrentUrlEquals(urlNavigatedTo);
     },
+
+    seeDocument(title, name, status, reason = '') {
```
Comment (swalker125) [inline e2e/steps_file.js]: if you put name = ' ' in the constructor, then in uploadDocuments_test it saves you passing in an empty string

File: e2e/pages/uploadDocuments/uploadDocuments.js (hunk 3 above)
Comment (geek-soft) [reply e2e/pages/uploadDocuments/uploadDocuments.js]: More `selectSocialWorkChronologyToFollow` ;)

File: e2e/paths/uploadDocuments_test.js [hunk 5]
```diff
@@ -7,19 +7,25 @@ Before((I, caseViewPage) => {
...
+  uploadDocumentsPage.socialWorkChronologyToFollow(config.testFile);
   uploadDocumentsPage.uploadSocialWorkStatement(config.testFile);
   uploadDocumentsPage.uploadSocialWorkAssessment(config.testFile);
   uploadDocumentsPage.uploadCarePlan(config.testFile);
   uploadDocumentsPage.uploadAdditionalDocuments(config.testFile);
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
   I.seeEventSubmissionConfirmation(config.applicationActions.uploadDocuments);
+  caseViewPage.selectTab('evidence');
```
Comment (geek-soft) [inline e2e/paths/uploadDocuments_test.js]: Why not to do `tabs.evidence`? Using text that must match property name makes it fragile.

File: e2e/paths/uploadDocuments_test.js (hunk 1 above)
Comment (kurt-uk) [reply e2e/paths/uploadDocuments_test.js]: changed

**Pull request merged**